README.md

```diff
@@ -375,36 +360,37 @@ To use Regula with Conftest:
 
 ### Directory structure
 
- -  `bin/`: the main Regula script that calls `terraform` & `opa`.
+ -  `bin/`: the main Regula script that, depending on the IaC platform, will call `opa` and `terraform` or convert CloudFormation YAML to JSON.
  -  `lib/`: the OPA library code to evaluate rules and mangle input.
- -  `rules/`: a collection of rules.  We may split this up further as the number
-    of rules increases.
+ -  `rules/`: a collection of rules.
  -  `examples/`: a collection of example rules that you can use as inspiration
      for your own rules.
  -  `scripts/`: scripts for development; currently only a script to generate
     test input.
  -  `tests/`:
       *  `tests/lib`: internal tests for the library.
       *  `tests/rules/`: tests for the various rules.
-      *  `tests/rules/inputs`: terraform files that can be used to generate Rego
+      *  `tests/rules/tf/<provider>/<service>/inputs`: tests for Terraform rules
+      *  `tests/rules/cfn/<service>/inputs`: tests for CloudFormation rules
+      *  `tests/rules/inputs`: CloudFormation and Terraform files that can be used to generate Rego
          files.
       *  `tests/examples/`: tests for the example rules.
       *  `tests/examples/inputs`: input files for the example rules.
 
 ### Adding a test
 
 If you would like to add a rule, we recommend starting with a test.
-Put your terraform code in a file in `tests/rules/<provider>/inputs`; for example
-[tests/rules/aws/inputs/kms\_rotate\_infra.tf](tests/rules/aws/inputs/kms_rotate_infra.tf).
+Put your code in a file in `tests/rules/tf/aws/kms/inputs`; for example
+[tests/rules/tf/aws/inputs/kms\_rotate\_infra.tf](tests/rules/tf/aws/inputs/kms_rotate_infra.tf).
 From this, you can generate a mock input by running:
 
     bash scripts/generate-test-inputs.sh
 
 The mock input will then be placed in a `.rego` file with the same name,
-in our case [tests/rules/aws/inputs/kms\_rotate\_infra.rego](tests/rules/aws/inputs/kms_rotate_infra.rego).
+in our case [tests/rules/tf/aws/inputs/kms\_rotate\_infra.rego](tests/rules/tf/aws/inputs/kms_rotate_infra.rego).
 
 Next, add the actual tests to a Rego file with the same name (appended with `_test` instead of `_infra`),
-but outside of the `inputs/` subdirectory.  Using this example, that would be [tests/rules/aws/kms\_rotate\_test.rego](tests/rules/aws/kms_rotate_test.rego).
+but outside of the `inputs/` subdirectory.  Using this example, that would be [tests/rules/tf/aws/kms\_rotate\_test.rego](tests/rules/tf/aws/kms_rotate_test.rego).
```

Comment:
Looks like the filename and path are both different now -- should be `tests/rules/tf/aws/kms/key_rotation_test.rego`

Reply:
Should we change line 392 as well to indicate that the test filenames don't have to be the same as the input filenames?

Reply:
Re: Line 392 - that's definitely true in general, but we have a CI check that enforces that there's a corresponding `_test` for each rule.